lib/schema-converters.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { iter } = require('metarhia-common');
+const { all: decorators } = require('./decorators');
+
+const defaultOptions = {
+  useEsm: false,
+};
+
+const createImport = name => `import { ${name} } from 'metaschema';`;
+const createRequire = name => `const { ${name} } = require('metaschema');`;
+
+const generateImports = (usedDecorators, useEsm) =>
+  // eslint-disable-next-line implicit-arrow-linebreak
+  `${(useEsm ? createImport : createRequire)('decorators')}\n` +
+  `const { ${usedDecorators.join(', ')} } = decorators;`;
+
+const toJs = (source, options = {}) => {
+  const { useEsm } = Object.assign({}, defaultOptions, options);
+  const usedDecorators = iter(Object.keys(decorators))
+    // eslint-disable-next-line no-useless-escape
+    .filter(name => source.match(`${name}\(.*\)`))
```

Comment:
The ESLint warning is legit and I don't quite understand why you disabled it since it can't give any false-positives. These should be `\\(` and `\\)`.